pkg/porter/mixins.go

```diff
@@ -62,6 +63,41 @@ func (p *Porter) ListMixins() ([]mixin.Metadata, error) {
 	return mixins, nil
 }
 
+// SearchMixins searches the internal remote mixins list according to the provided options
+func (p *Porter) SearchMixins(opts mixin.SearchOptions) error {
```

Comment:
Can you split this function into two functions? One that handles doing the search and other than applies the printer options and does the printing?

As always I ask for that because it helps people reuse code (you can't reuse code that prints to the console) and test smaller units of work.

Reply:
Good call; should be split up now.